corehq/form_processor/models.py

```diff
@@ -1203,15 +1203,16 @@ class LedgerValue(DisabledDbMixin, models.Model, TrackRelatedChanges, AbstractLe
     """
     objects = RestrictedManager()
 
-    # domain not included and assumed to be accessed through the foreign key to the case table. legit?
+    domain = models.CharField(max_length=255, null=False, default=None)
     case_id = models.CharField(max_length=255, db_index=True, default=None)  # remove foreign key until we're sharding this
+    location_id = models.CharField(max_length=255, null=True, default=None)
```

Comment:
yea, it is for reporting so that you can do `SQLLocation.by_location_id(location_id)` which is used in quite a lot of places. 

If we dump to ES we'd still need to do a lookup for `SQLLocation`.

Nearly done with converting another report - can re-evaluate then on the ES option.